src/cnaas_nms/devicehandler/sync_devices.py

```diff
@@ -271,7 +271,7 @@ def populate_device_vars(
                     ifindexnum = 0
                 if "ifclass" not in intf:
                     continue
-                extra_keys = ["aggregate_id", "enabled", "cli_append_str", "metric", "mtu", "tags"]
+                extra_keys = ["aggregate_id", "enabled", "cli_append_str", "metric", "mtu", "tags", "patch_postion"]
```

Comment:
this is for dist/core devices, if you want to have an option for patch_position on those devices as well you need to make some changes in settings_fields.py as well